src/ChickenAPI.Game/Chat/Events/ChatHeroEvent.cs

```diff
@@ -0,0 +1,9 @@
+﻿using ChickenAPI.Game._Events;
+
+namespace ChickenAPI.Game.Chat.Events
+{
+    public class ChatHeroEvent : GameEntityEvent
```

Comment:
All ChickenAPI related code should be written on ChickenAPI's repository, not here, #6 is going to remove ChickenAPI of the repository